wp-config.php

```diff
@@ -8,48 +8,46 @@
  * https://pantheon.io/docs
  */
 
+/**
+ * Pantheon platform settings. Everything you need should already be set.
+ */
+if (file_exists(dirname(__FILE__) . '/wp-config-pantheon.php') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
+	require_once(dirname(__FILE__) . '/wp-config-pantheon.php');
+
 /**
  * Local configuration information.
  *
  * If you are working in a local/desktop development environment and want to
  * keep your config separate, we recommend using a 'wp-config-local.php' file,
  * which you should also make sure you .gitignore.
  */
-if (file_exists(dirname(__FILE__) . '/wp-config-local.php') && !isset($_ENV['PANTHEON_ENVIRONMENT'])):
-  # IMPORTANT: ensure your local config does not include wp-settings.php
-  require_once(dirname(__FILE__) . '/wp-config-local.php');
+} elseif (file_exists(dirname(__FILE__) . '/wp-config-local.php') && !isset($_ENV['PANTHEON_ENVIRONMENT'])){
+	# IMPORTANT: ensure your local config does not include wp-settings.php
+	require_once(dirname(__FILE__) . '/wp-config-local.php');
 
 /**
- * Pantheon platform settings. Everything you need should already be set.
+ * This block will be executed if you have NO wp-config-local.php and you
+ * are NOT running on Pantheon. Insert alternate config here if necessary.
```

Comment:
The sentence could be restructured to match the actual order the checks are done. "This block will be executed if you are NOT running on Pantheon and have NO wp-config-local.php."

Reply:
Good call. Updated.
```
 This block will be executed if you are NOT running on Pantheon and have NO
 wp-config-local.php. Insert alternate config here if necessary.
```